go/vt/vttablet/tabletserver/planbuilder/plan.go

```diff
@@ -194,6 +194,8 @@ func Build(statement sqlparser.Statement, tables map[string]*schema.Table, isRes
 		plan, err = &Plan{PlanID: PlanRelease}, nil
 	case *sqlparser.SRollback:
 		plan, err = &Plan{PlanID: PlanSRollback}, nil
+	case *sqlparser.Load:
+		plan, err = &Plan{PlanID: PlanDDL}, nil
```

Comment:
Load data is should not have PlanID as PlanDDL. It should have its own PlanID.

Reply:
I think should be able to use `PlanOtherAdmin`. The query still says `LOAD DATA`, so I don't feel like it needs it's own ID.

It's clearly not DDL

Reply:
Done.